packages/devtools_app/lib/src/screens/debugger/debugger_controller.dart

```diff
@@ -912,7 +918,7 @@ class DebuggerController extends DisposableController
 
     if (frame?.scriptRef != null) {
       showScriptLocation(
-          ScriptLocation(frame.scriptRef, location: frame.position));
+          ScriptLocation(frame!.scriptRef!, location: frame.position));
```

Comment:
hoist frame?.scriptRef into a local. Then when you check that the local var is not null, you can use the local here where it will have been promoted to a non-nullable type